ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ImmutableDB/Model.hs

```diff
@@ -94,6 +98,14 @@ dbmBlobs dbm@DBModel {..} = foldr add ebbs (zip (map SlotNo [0..]) dbmChain)
          & Map.mapKeysMonotonic (\epochSlot ->
              (epochSlot, epochSlotToSlot dbm epochSlot))
 
+dbmTipBlock :: DBModel hash -> Maybe TestBlock
```

Comment:
This deserves a comment :-o

Reply:
Better yet: perhaps it would not be too difficult to main `DBModel` polymorphic in `a`.

Reply:
I have created #1151 and I will add a TODO referring to it in the code.